client/main.lua

```diff
@@ -102,6 +102,64 @@ local function GetClosestCheckpoint()
     RaceData.ClosestCheckpoint = current
 end
 
+local function SetupPiles()
+    for k, v in pairs(CreatorData.Checkpoints) do
+        if CreatorData.Checkpoints[k].pileleft == nil then
+            ClearAreaOfObjects(v.offset.left.x, v.offset.left.y, v.offset.left.z, 50.0, 0)
+            CreatorData.Checkpoints[k].pileleft = CreateObject(`prop_offroad_tyres02`, v.offset.left.x, v.offset.left.y, v.offset.left.z, 0, 0, 0)
```

Comment:
The last 3 arguments need to be booleans, in this case all false

Reply:
Fixed these and other linting issues in latest push.